bin/pull-image

```diff
@@ -1,13 +1,20 @@
 #! /usr/bin/env bash
 
-# DOC: Pull the latest development environment docker image from Docker Hub unless USE_LOCAL_CIVIFORM is set.
+# DOC: Pull the latest development environment docker image from Docker Hub
+# DOC: unless USE_LOCAL_CIVIFORM is set. May specify a specific what to pull
```

Comment:
"specific what" reads weird.

also include "single" since you can't specify 2 things.

Reply:
wups, typo. fixed